lib/ws-channel.js

```diff
@@ -126,10 +138,13 @@ WebsocketChannel.prototype._detach = function() {
 
 
 WebsocketChannel.prototype.close = function(callback) {
+  this._debug('close');
+
   if (this._websocket) {
     this._unscheduleSendAck();
     this._flushSendQueue();
     this._closeWebsocket(this._websocket);
+    this._websocket = null;
```

Comment:
@piscisaureus every line of this PR is debug output... except for this one.

It looks to me that after close, `_websocket` should be nulled, as it is a few lines up at the end of `_detach()`. Seem reasonable?